spec/abstract.md

```diff
@@ -3,27 +3,34 @@
 This document specifies a [DID
 Method](https://w3c-ccg.github.io/did-spec/#specific-did-method-schemes),
 `did:webs`, that is web-based but also highly and innovatively secure. Like its
-predecessor, [`did:web`](https://w3c-ccg.github.io/did-method-web/), the
+interoperable cousin, [`did:web`](https://w3c-ccg.github.io/did-method-web/), the
 `did:webs` method uses websites and popular, mature tooling to publish DIDs and
 make them discoverable. Unlike `did:web`, this method's trust is not rooted in
-DNS, webmasters, X509, and certificate authorities. Instead, it derives from
-meticulous usage of cryptographic keys by those who control identifiers.
+DNS, webmasters, X509, and certificate authorities. Instead, it uses [[ref:
+KERI]] to provide a secure chain of cryptographic keys events by those who
+control the identifier.
 
-The `did:webs` method does not need blockchains for trust. However, it can
-reference arbitrary blockchains as an extra, optional publication mechanism.
-This offers an interoperability bridge from (or between) blockchain ecosystems.
-Also, without directly supporting environments where the web is not practical
-(e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a foundation that fully
-supports those environments, making future interop of identifiers between web
-and non-web an easy step.
+The `did:webs` method does not need blockchains for trust. However, its use of
+KERI allows for the referencing of arbitrary blockchains as an extra, optional
+publication mechanism. This offers an interoperability bridge from (or between)
+blockchain ecosystems. Also, without directly supporting environments where the
+web is not practical (e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a
+foundation that fully supports those environments, making future interop of
+identifiers between web and non-web an easy step.
+
+`did:webs` takes advantage of its web basis to introduce two conventions that
```

Comment:
I like the whois idea. However, I don't think this is the right place to introduce it, and I don't think it should be viewed as a central feature of the DID method, but rather as a logical extension to it. The DID method should introduce the idea that any path "below" the DID requires signed files, and cite "whois" as an example. Then an appendix or a separate doc should explain the meaning of whois. Putting this in the abstract surfaces it too early. Again, I'm not unenthused. I just think it's not the right place/context to bring it up.

Reply:
I’d like to see it up front, but have removed it.  I’ll add it elsewhere in the spec.